llvm/lib/Target/RISCV/MCTargetDesc/RISCVFixupKinds.h

```diff
@@ -71,6 +71,18 @@ enum Fixups {
   // Used to generate an R_RISCV_ALIGN relocation, which indicates the linker
   // should fixup the alignment after linker relaxation.
   fixup_riscv_align,
+  // 20-bit fixup corresponding to %tlsdesc_hi(foo) for instructions like
+  // auipc
+  fixup_riscv_tlsdesc_hi20,
+  // 12-bit fixup corresponding to %tlsdesc_load_lo(foo) for instructions like
+  // addi
```

Comment:
Should this say load instead of addi?

Reply:
Yes, I've fixed it now to use `lw`, but maybe it should just read `... for load instructions` instead? WDYT?